index.html

```diff
@@ -28,46 +28,54 @@
   <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap" rel="stylesheet">
 </head>
 <body>
-  <div class="row background-header height-header align-header column-full">
-    <header>
-      Code Journal
-    </header>
+  <div class="row background-header height-header align-header">
```

Comment:
This still needs to be in a container. Make the container a child of the header, and put the text in a headline element (h1 - h6)

```
header - apply background color here
  container
    row
      column-full
        h?
```